src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/simulering/SimuleringUtil.kt

```diff
@@ -45,9 +45,19 @@ fun lagSimuleringsoppsummering(
         forfallsdatoNestePeriode = nestePeriode?.forfallsdato,
         tidSimuleringHentet = tidSimuleringHentet,
         tomSisteUtbetaling = tomSisteUtbetaling,
+        sumManuellePosteringer = summerManuellePosteringer(detaljertSimuleringResultat)
     )
 }
 
+fun summerManuellePosteringer(detaljertSimuleringResultat: DetaljertSimuleringResultat): BigDecimal {
+    return detaljertSimuleringResultat.simuleringMottaker
+        .flatMap { simuleringMottaker -> simuleringMottaker.simulertPostering }
+        .filter { simulertPostering ->
+            simulertPostering.fagOmrådeKode.name == FagOmrådeKode.ENSLIG_FORSØRGER_OVERGANGSSTØNAD_MANUELL_POSTERING.name
```

Comment:
Denne funker for overgangsstønad, men bør vi sjekke for barnetilsyn også?
Den heter `MEFBT` for ny løsning og `MIT30BT` for infotrygd.
Ser ut som den som brukes her (`MEFOGNY`) gjelder for infotrygd, men kan vi få manuelle posteringer på `MEFOG` også? 🤔

Reply:
MEFOGNY er den eneste fagområdekoden som er i kontrakter ser det ut som. Bør de andre kodene legges til, i så fall ?

Reply:
De kan jo dukke opp, men vet ikke - hvis vi ikke har fått inn noe til nå, så er det kanskje ikke så stor sannsynlighet for at det dukker opp så mye fremover heller. 

Sjelden man opphører/endrer barnetilsyn mange år tilbake i tid, vil jeg tro.

Reply:
Men det kan komme noe gøy i fremtiden i EØS-saker da - så MEFBT er nok ikke så dumt å legge inn, når jeg tenker meg om... Men den infotrygdkoden er vel heller usannsynlig at dukker opp.